lib/api/_loaders/element-global.js

```diff
@@ -156,6 +156,12 @@ class ElementGlobal {
     if (locator instanceof Element) {
       this.locator = locator;
     } else {
+      if (this.client && this.client.locateStrategy){
+        locator = {
+          value: locator,
+          using: this.client.locateStrategy
+        };
+      }
```

Comment:
The case where `use_xpath:true` but we want to use css selector in a command like `element({selector: 'input', using: 'css selector'})` is not handled properly.